elements/pfe-label/pfe-label.ts

```diff
@@ -118,6 +149,12 @@ export class PfeLabel extends BaseLabel {
     `;
   }
 
+  willUpdate() {
+    super.willUpdate();
+    const pfeClasses = { compact: this.compact, truncated: this.truncated };
+    this.classes = { ...this.classes, ...pfeClasses };
+  }
+
```

Comment:
@bennypowers curious to hear your opinion on this approach of adding classes to the class map from the child class.

Reply:
if it was up to me i'd rather just do it in `render()`, but this is fine